### ISSUE TITLE
wipe remembered device 

### ISSUE THREAD

**Issue by mroz22** [trezor/trezor-suite#1766]
### with nothing remembered

before wipe
![image](https://user-images.githubusercontent.com/30367552/81499761-2468fd00-92ce-11ea-8511-a7c30728ce2d.png)

after wipe. not possible to cancel. after disconnect or reload device is not paired. works fine.
![image](https://user-images.githubusercontent.com/30367552/81499858-b7a23280-92ce-11ea-9eff-346a75b945b6.png)

### with remembered wallets

state of affairs right before wipe

![image](https://user-images.githubusercontent.com/30367552/81499653-5af24800-92cd-11ea-9241-a00e97a2fa43.png)

after wipe

![image](https://user-images.githubusercontent.com/30367552/81499858-b7a23280-92ce-11ea-9eff-346a75b945b6.png)

after disconnecting device switch-device modal still looks like this. that is expected 

![image](https://user-images.githubusercontent.com/30367552/81499653-5af24800-92cd-11ea-9241-a00e97a2fa43.png)

what happens when I connect the device again? lets see. right after it is connected nothing happens. it is not paired yet.

In switch-device modal I may click on "Find Trezor" button 

![image](https://user-images.githubusercontent.com/30367552/81500028-b291b300-92cf-11ea-94ed-023ad7863065.png)

After I do so and pair device: 

![image](https://user-images.githubusercontent.com/30367552/81500052-cf2deb00-92cf-11ea-87f6-7a2dff38a250.png)

- [ ] question: here I would expect another device to appear alongside with the device previously saved and already wiped. 
- [ ] question: also, shouldnt wipe wipe everything? Or is it expected usecase that user wipes device but wants to keep it saved in read only mode? 

then I click on the solve issue button and get this:

![image](https://user-images.githubusercontent.com/30367552/81499675-78bfad00-92cd-11ea-9001-f392b21a61b4.png)

- [ ] question: should swith device button here be either shown only if there are multiple physical devices or kept as it is and reworded to "switch wallet"? 

So, I pass through onboarding and switch device modal looks like this again

![image](https://user-images.githubusercontent.com/30367552/81499653-5af24800-92cd-11ea-9241-a00e97a2fa43.png)

but of course saved addresses and maybe transactions are no longer valid. this does not happen only with webusb but also with bridge transport

**Comment by tsusanka** [comment]
@STew790 please retest if this is still relevant.

**Comment by STew790** [comment]
I tested both WebUSB as well as Bridge transport and I could not reproduce this issue.

When I wiped the device, the Suite correctly offered me to begin the onboarding and after the onboarding, there were no "remembered wallets" in the wallet choosing modal.

Although I encountered a different issue when I was trying to reproduce this with WebUSB, the device after wiping could not begin the onboarding right away, even though the Suite sees it connected, but I had to reconnect the device to be able to begin the onboarding. I will create a separate issue for this.
